pkg/hive/cell/health.go

```diff
@@ -110,32 +158,49 @@ func (s *Status) String() string {
 	} else {
 		sinceLast = time.Since(s.LastUpdated).String() + " ago"
```

Comment:
This seems to be a rendering decision? Perhaps we should let consumers defined how since should be formatted.

Reply:
Yeah I agree, but let's try to get all the formatting in one place, and eventually move this code elsewhere (i.e. we can implement different formatting modes via aliases on *UpdateNode).